README.md

```diff
@@ -1,4 +1,6 @@
 # Numix Core
+[![Build Status](https://travis-ci.org/numixproject/numix-core.svg?branch=master)](https://travis-ci.org/numixproject/numix-core)
```

Comment:
@Foggalong Don't worry about the unknown status here, as the badge shows the status of the master branch, and as the master branch doesn't contain any .travis.yml file it shows this badge; but once the merge is done, we will see a beautiful green one :)